app/docs/0.10.x/proxy.md

```diff
@@ -138,15 +125,13 @@ HTTP/1.1 201 Created
 ```
 
 This request instructs Kong to register an API named "my-api", reachable at
-"http://my-api.com". It also specifies various routing properties. In reality,
-**only one of** `hosts`, `uris` and `methods` is actually required. More on this
-later.
+"http://my-api.com". It also specifies various routing properties, though note that **only one of** `hosts`, `uris` and `methods` is  required. 
 
 Adding such an API would mean that you configured Kong to proxy all incoming 
-requests matching `hosts`, `uris`, and `methods` to `http://my-api.com`.
-Kong being a transparent proxy, it will forward the request to your upstream
-service - practically - untouched (exception made of various headers such as
-`Connection`, or in case a plugin performs some transformations to it).
+requests matching the specified `hosts`, `uris`, and `methods` to `http://my-api.com`.
+Kong is a transparent proxy, and it will forward the request to your upstream
+service untouched, with the exception of the addition of various headers such as
```

Comment:
can remove `of the addition` as now we repeat `of` twice in a single sentence in close proximity and we reduce sentence complexity for brevity